v03_pipeline/lib/tasks/write_imported_callset.py

```diff
@@ -62,50 +66,64 @@ def requires(self) -> list[luigi.Task]:
         ]
 
     def create_table(self) -> hl.MatrixTable:
-        mt = import_callset(
-            self.callset_path,
-            self.reference_genome,
-            self.dataset_type,
-        )
-        filters_path = None
-        if (
-            Env.EXPECT_WES_FILTERS
-            and not self.skip_expect_filters
-            and self.dataset_type.expect_filters(
-                self.sample_type,
+        if self.clone(WriteValidationErrorsForRunTask).complete():
+            raise SeqrValidationError(
+                self.clone(WriteValidationErrorsForRunTask).to_single_error_message(),
             )
-        ):
-            filters_path = valid_filters_path(
-                self.dataset_type,
-                self.sample_type,
+        try:
```

Comment:
this feels like a very large try/except block given that AFAIK there is only one function call in this block thats capable of generating the exception thats being caught. I think it would be more readable to just wrap that one call in the try/except

Reply:
I think there are four functions that can throw validation errors (`import_vcf`, `select_relevant_fields` (will throw if a field we're expecting is missing), `validate_imported_field_types`, and `split_multi_hts`).  Only one is explicit though, so a comment is worthwhile?  Or I can at least improve the function names to clarify that they're capable of throwing an exception (that feels a little `Java` to me though).

Reply:
they're currently decorated functions:

```
@validated_hl_function(...)
def import_vcf()
```

renaming to 

```
@validated_hl_function(...)
def import_valid_vcf()
```

and having the function name be misleading.  Or doing:

```
import_valid_vcf = validated_hl_function(import_vcf)
```

seems weird to me.  

[Renaming](https://stackoverflow.com/questions/10874432/possible-to-change-function-name-in-definition) within the decorator feels inappropriate.

Reply:
I added some helpful comments to these functions to make it more clear.